documents/Libraries/stdlib/gen-glsl/mx_noise2d_color2.glsl

```diff
@@ -1,7 +1,7 @@
-#include "stdlib/sx-glsl/sx_bits_to_01"
+#include "stdlib/gen-glsl/mx_bits_to_01"
```

Comment:
This doesn't look right, there is no such file. I think the include should be "stdlib/gen-glsl/lib/mx_noise.glsl".

Reply:
I was confused by this as well. I'll check.